ibis/expr/api.py

```diff
@@ -14,53 +14,31 @@
 
 from __future__ import print_function
 
+import six
+import toolz
 import warnings
 import operator
 import datetime
 import functools
 import collections
 
-import six
-import toolz
+import ibis.util as util
+import ibis.common as com
+import ibis.expr.types as ir
+import ibis.expr.rules as rlz
+import ibis.expr.schema as sch
+import ibis.expr.analysis as _L
+import ibis.expr.datatypes as dt
+import ibis.expr.analytics as _analytics
+import ibis.expr.operations as ops
```

Comment:
Is there any reason to change the `_ops` and `_com` imports? The changes in this file are rather noisy because it looks like most of them just renaming `_ops` to `ops` and `_com` to `com`. The reason these are prefixed with an underscore is to hide them from a star import.

Reply:
Api has `__all__` defined - which protects against star import, and I thought `ops` is cleaner.